crates/client/sync/src/l2.rs

```diff
@@ -381,6 +383,12 @@ async fn update_sync_metrics(
     block_metrics.l1_gas_price_wei.set(f64::from_u128(block_header.l1_gas_price.eth_l1_gas_price).unwrap_or(0f64));
     block_metrics.l1_gas_price_strk.set(f64::from_u128(block_header.l1_gas_price.strk_l1_gas_price).unwrap_or(0f64));
 
+    if block_number % 1000 == 0 {
```

Comment:
avoid magic number and replace `1000` by a constant eg. `STORAGE_SIZE_UPDATE_STEP`

Reply:
Please go back to your organization!